controllers/suite_test.go

```diff
@@ -317,7 +317,7 @@ func setUpTestProxy() {
 	}
 
 	err = k8sClient.Create(ctx, &testProxyDeployment)
-	Expect(err != nil || errors.IsConflict(err))
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
You are right, this seems wrong, but the conflict errors should be ignored:
```suggestion
	Expect(err == nil || errors.IsConflict(err))
```

Reply:
+1 (I shall add `.To(BeTrue())` assertion)

Reply:
Yes, all those occurrences are missing the assertion.